content/en/infrastructure/containers/orchestrator_explorer.md

```diff
@@ -161,6 +161,7 @@ There are multiple types of terms available:
 | **Annotations**: Extracted from [a resource's metadata][26]. They are generally used to support tooling that aid in cluster management. | `annotation#checksum/configmap:a1bc23d4` |
 | **Metrics**: Added to workload resources (pods, deployments, etc.). You can find resources based on their utilization. To see what metrics are supported, see [Resource Utilization Filters](#resource-utilization-filters). | `metric#cpu_usage_pct_limits_avg15:>80%` |
 | **String matching**: Supported by some specific resource attributes, see below.<br>*Note: string matching does not use the key-value format, and you cannot specify the attribute to match on.* | `"10.132.6.23"` (IP)<br>`"9cb4b43f-8dc1-4a0e"` (UID)<br>`web-api-3` (Name) |
+| **Fields**: Extracted from [a resource's metadata][27]. They are essentially resource filters. | `field#metadata.creationTimestamp:>=4wk field#metadata.deletionTimestamp:<=1hr` |
```

Comment:
We added the `field#` prefix to allow users to filter on manifest/YAML properties. Using the same path that they already know from the manifeststructure.

Today we only support `metadata.creationTimestamp` and `metadata.deletionTimestamp` but we could allow to filter on other fields like `field#status.startTime`, `field#spec.priority`, etc.  So I think the copy here is not good, it should not say that it's limited to the resource's metadata.

Reply:
Thanks for the review, would this be better?

```suggestion
| **Fields**: Extracted from a resource's manifest fields. They are essentially resource filters. | `field#metadata.creationTimestamp:>=4wk field#metadata.deletionTimestamp:<=1hr` |
```

Reply:
I'll leave this to better english speakers but yes something like that would work better for me

Reply:
For verbiage, I'd simplify it slightly and just say:

> Extracted from a resource's manifest.

I would break the example up into two lines, like the other rows
```
`field#metadata.creationTimestamp:>=4wk`<br>`field#metadata.deletionTimestamp:<=1hr`
```